css-paint-api/background-image-alpha.html

```diff
@@ -21,7 +21,7 @@
 }
 </style>
 
-<script src="../common/css-paint-tests.js"></script>
+<script src="/common/css-paint-tests.js"></script>
```

Comment:
Do the tests still work, or do they all need to include /common/reftest-wait.js?